### PR TITLE
Remove a confusing sentence from `about-code-owners.md`

### PR DESCRIPTION
**Pull request by boblail**
The CODEOWNERS docs start with the statement:

> People with admin or owner permissions can set up a CODEOWNERS file in a repository.

However, this is confusing because people with write permissions can create and edit the `CODEOWNERS` file itself whereas admin/owner permissions are required to _enforce_ reviews from code owners with branch protection settings.

See this discussion: [About CODEOWNERS - User with Write permissions also can setup?](https://github.com/orgs/community/discussions/36497#discussioncomment-3901570)

I'm proposing to remove the sentence because the doc already explains how to enforce reviews from code owners in this paragraph below:

> When someone with admin or owner permissions has enabled required reviews, they also can optionally require approval from a code owner before the author can merge a pull request in the repository. For more information, see "[About protected branches.](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#require-pull-request-reviews-before-merging)"

* * *

[Preview](https://docs-25692-f98a3d.preview.ghdocs.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)

* * *

### Why:

I actually took that sentence at face value for a while and _didn't consider using CODEOWNERS_ at my company in a context where it wouldn't have worked for only repo admins to be able to edit the file!

Closes: 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@boblail Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by sophietheking** [comment]
Hi @boblail. Thanks for opening this PR and contributing to our Docs. Would it work to edit the permission statement instead of remove it? Maybe something like:

> People with admin or maintain permissions can require a CODEOWNERS file in a repository. People with write, admin, and maintain permissions can create or edit the CODEOWNERS file.

**Comment by boblail** [comment]
@sophietheking, I'll just note that both those sentences _do_ show up in the doc. The very next sentence is:

> The people you choose as code owners must have write permissions for the repository.

and the third paragraph starts with

> When someone with admin or owner permissions has enabled required reviews, they also can optionally require approval from a code owner before the author can merge a pull request in the repository.

**Comment by boblail** [comment]
@sophietheking, sounds good! I've updated the PR. I called out that

- `write` is required to add or edit the file
- `write` is required to be listed in the file as code owners
- `admin`/`owner` is required to enforce the file in branch protection settings

Looks like:

> **Who can use this feature**
> People with write permissions for the repository can create or edit the CODEOWNERS file and can be listed as code owners in it. People with admin or owner permissions can require that pull requests by approved by code owners before they can be merged.

File: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md
Comment (sophietheking) [inline content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md:4]:
```suggestion
permissions: 'People with write permissions for the repository can create or edit the CODEOWNERS file and be listed as code owners. People with admin or owner permissions can require that pull requests have to be approved by code owners before they can be merged.'
```

**Review by sophietheking — COMMENTED** [review]
I left a small suggestion to change the wording slightly. I think this change makes the requirements a lot clearer, thanks for working on this with me! 😄 

I'll get this merged if you're happy with my suggestion. ✨

**Review by sophietheking — DISMISSED** [review]
Approving! ✅

**Comment by boblail** [comment]
@sophietheking, I accepted the suggestion and that dismissed your approval. Feel free to merge when you're ready! 🙂 🚀

File: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md
Comment (woottpp) [inline content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.md:21]:
```suggestion
## About code owners
```

**Review by sophietheking — APPROVED** [review]
Looking good! 🙌

**Pull request merged**
